documentation/site/themes/tbds/layouts/_default/list.html

```diff
@@ -0,0 +1,19 @@
+{{ define "main" }}
+  <h2>
```

Comment:
Maybe swap the `h2` and `h1` in order? Would be odd to assistive technology to work backward in the tree.

Reply:
Actually, I'm going to remove that `h1` from the layout.